docs/manual/common/welcome/WhereToStart.md

```diff
@@ -0,0 +1,14 @@
+# Where to begin
```

Comment:
My comment here is a duplicate of @TimMoore's https://github.com/lagom/lagom/pull/516/files/ae91d6fc42415fb0f52112aa2dff1df17fe8f24b#r102888983